frontend/js/marc-record-merge-postmerge-service.js

```diff
@@ -447,6 +462,31 @@ export function removeExtra035aFromMerged(preferredRecord, otherRecord, mergedRe
   };
 }
 
+export function add080VersionCode (preferredRecord, otherRecord, mergedRecordParam) {
+  const mergedRecord = new MarcRecord(mergedRecordParam);
+
+  mergedRecord.fields.filter(field => field.tag === '080')
+    .forEach(field => {
+      let hasCode9FKeep = false;
+      let hasCode2 = false;
+      field.subfields.forEach(sub => {
+        if (sub.code === '2') {
+          hasCode2 = true;
+        }
+        if (sub.code === '9' && sub.value === 'FENNI<KEEP>') {
+          hasCode9FKeep = true;
+        }
+      });
+      if (!hasCode2 && hasCode9FKeep) {
+        field.subfields.push({ code: '2', value: '1974/fin/fennica' });
+        field.subfields.sort(sortSubfields);
```

Comment:
Subfields shouldn't be sorted but the `2` subfield should be inserted in correct numerical order if there are other numeric subfields.

Do it like this:
- If there are no numeric subfield push to the end of the list
- Otherwise insert into correct numerical index in relation to other numeric subfields.